README.md

```diff
@@ -397,6 +397,8 @@ TF_VAR_NOTIFICATION_HUB_GCM_KEY="XXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
 TF_VAR_NOTIFICATION_HUB_APNS_KEY_ID="XXXXXXXXXXX"
 TF_VAR_NOTIFICATION_HUB_APNS_KEY="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
 
+# Webhook API backend (contains secret token)
+TF_VAR_WEBHOOK_CHANNEL_URL_TOKEN="XXXXXXXXXXXXXXXXXXXXXXXXXXX"
```

Comment:
is this the url that contains the token or the token that will be appended to the url? the name `URL_TOKEN` suggests it's just the token, but the comment says otherwise